WooCommerce/Classes/ServiceLocator/ServiceLocator.swift

```diff
@@ -0,0 +1,26 @@
+
+import Foundation
+
+final class ServiceLocator {
+    private static var _analytics: Analytics = WooAnalytics(analyticsProvider: TracksProvider())
+    //private static var _storesManager: StoresManager = DefaultStoresManager()
```

Comment:
Looks like storesManager is next in line to be added? You can always create a git stash with the StoresManager changes, so you don't forget where you left off. Then you can delete this commented out code when it's time to merge.

Reply:
Right, thanks! I was thinking that it'd probably be easier to do the refactoring in stages. First, analytics, then stores, and finally app delegate. If only, to keep the diff manageable